lib/get_ast.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
+const {Parser} = require("acorn")
+
+const sourceType = process.argv[2]
+const src = fs.readFileSync(process.argv[3], 'utf8')
+const tree = Parser.parse(src, {'locations': true, 'sourceType': sourceType,
+                                'ecmaVersion': '2020'})
+const classFields = require('acorn-class-fields');
```

Comment:
```suggestion
```

The `acorn-class-fields` package is not used in this script. Can you remove this line?
It causes an error saying "Cannot find module 'acorn-class-fields'" if it is not installed.

Reply:
Fixed. Thank you